code/modules/projectiles/guns/rifles.dm

```diff
@@ -2721,10 +2721,6 @@
 		/obj/item/attachable/foldable/bipod,
 		/obj/item/weapon/gun/pistol/plasma_pistol,
 		/obj/item/weapon/gun/shotgun/combat/masterkey,
-		/obj/item/weapon/gun/flamer/mini_flamer,
-		/obj/item/weapon/gun/grenade_launcher/underslung,
-		/obj/item/weapon/gun/rifle/pepperball/pepperball_mini,
-		/obj/item/weapon/gun/grenade_launcher/underslung/mpi,
```

Comment:
I was only questioning the mpi UGL btw, since thats just a normal UGL but with a custom sprite specifically to fit the MPI, and it is unremovable.